joomla/joomla-cms
# [4.0] Add open new window tooltip

Continuation of #27993 .

### Summary of Changes
Add "Open new window" tooltip.


### Testing Instructions
Go to System > Administrator Template Styles
Click Options button
Enable `Preview Module Positions`
Click Save & Close button
Hover mouse over icon under Preview column.
See tooltip


### Actual result BEFORE applying this Pull Request
No tooltip
